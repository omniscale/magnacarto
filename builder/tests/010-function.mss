@foo: #a03;
@bar: lighten(rgba(100, 50, 75, 150), 20);

#func        { line-width: 1; line-color: fadeout(@foo, 10%); }
#funcnested  { line-width: 1; line-color: fadeout(lighten(@foo, 10), 20); }
#funcfunc    { line-width: 1; line-color: fadeout(@bar, 12); }

