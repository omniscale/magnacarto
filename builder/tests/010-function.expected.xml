<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="func" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke="rgba(170, 0, 51, 0.90000)" stroke-width="1"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="funcnested" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke="rgba(221, 0, 66, 0.80000)" stroke-width="1"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="funcfunc" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke="rgba(168, 84, 126, 0.46824)" stroke-width="1"></LineSymbolizer>
    </Rule>
  </Style>
  <Layer name="func" srs="" status="off">
    <StyleName>func</StyleName>
  </Layer>
  <Layer name="funcnested" srs="" status="off">
    <StyleName>funcnested</StyleName>
  </Layer>
  <Layer name="funcfunc" srs="" status="off">
    <StyleName>funcfunc</StyleName>
  </Layer>
</Map>